packages/appeals-service-api/src/routes/v2/token/controller.js

```diff
@@ -65,6 +65,7 @@ async function tokenPostV2(req, res) {
 	// user has confirmed email for first time so set enrolled flag
 	if (!user.isEnrolled) {
 		user.isEnrolled = true;
+		sendConfirmRegistrationEmailToAppellant(user.email, user.id, user.email);
```

Comment:
We only want to send this is `enrol-users` flag is active - is that checked somewhere?

Also I think you're missing an `await` here.